src/main/resources/static/index.html

```diff
@@ -15,11 +15,14 @@
 <script src="modules/auth/AuthService.js"></script>
 <script src="modules/auth/AuthCtrl.js"></script>
 
+
 <!-- Bootstrap core JavaScript
 ================================================== -->
 <!-- Placed at the end of the document so the pages load faster -->
 <script src="js/vendor/jquery/jquery.min.js"></script>
 <script src="js/vendor/bootstrap/bootstrap.min.js"></script>
```

Comment:
Nico, this is something that I have added with the design on the last commit pushed to master. I will remove. We can talk about it.